articles/connections/social/oauth2.md

```diff
@@ -35,39 +36,42 @@ curl -H "Content-Type: application/json" -H 'Authorization: Bearer {YOUR_GLOBAL_
 
 The key parameters for a connection are:
 
-* **name**: this is how the connection can be referenced later on in Auth0 or in your app.
-* **strategy**: this should always be __oauth2__. It defines the protocol implemented by the provider.
+* **name**: how the connection will be referenced in Auth0 or in your app.
+* **strategy**: defines the protocol implemented by the provider. This should always be `oauth2`.
 
-The __options__ object:
+The **options** object:
 
-* **client_id** and **client_secret** must be obtained from your provider.
-* **authorizationURL**: this is the endpoint where the transaction starts. Usually looks like: `https://your.oauth2.server/oauth2/authorize`.
-* **tokenURL**: Auth0 will use this URL to exchange the `code` for an `access_token`. Looks like: `https://your.oauth2.server/oauth2/token`.
-* **scope**: scope parameters that you want to request consent for (e.g. `profile`, etc.).
-* **fetchUserProfile**: Auth0 allows you to define a custom script that returns a JSON object with user info. What you do in the script is up to you.
+* **client_id** and **client_secret**: obtained from your provider.
+* **authorizationURL**: the URL where the transaction begins. Usually similar to `https://your.oauth2.server/oauth2/authorize`.
+* **tokenURL**: the URL Auth0 will use to exchange the `code` for an `access_token`. Usually similar to `https://your.oauth2.server/oauth2/token`.
+* **scope**: the scope parameters that you want to request consent for (e.g. `profile`, etc.).
+* **fetchUserProfile**: a custom script that returns a JSON object with user info.
 
-The script will have the following signature:
+### Custom script
 
-```
-function(ctx,access_token,callback){
+A custom script will have the following signature:
+
+```js
+function(ctx, access_token, callback){
 
   var profile = {
     user_id: '123',
-    goiven_name: 'Eugenio',
+    given_name: 'Eugenio',
     family_name: 'Pace',
     email: 'eugenio@mail.com'
   };
 
   callback(null, profile);
 }
-
 ```
 
-The `access_token` parameter will often be used for authenticating requests to the provider's API.
+The `access_token` parameter can be used for authenticating requests to the provider's API.
 
-For example, retrieving the user profile from __GitHub__ would look like:
+**NOTE:** We recommend using the field names from the [normalized profile](/user-profile#normalized-user-profile).
 
-```
+For example, the following code will retrieve the user profile from **GitHub**:
+
+```js
 function(ctx, access_token, callback) {
```

Comment:
In all the examples at https://auth0.com/docs/oauth2-examples, the syntax is `function(accessToken, ctx, cb)` - that is accessToken is the first parameter, not the second.